ocean_lib/ocean/ocean_assets.py

```diff
@@ -314,7 +315,7 @@ def create(
                 publisher_wallet,
             )
             if not ddo_registry.verify_tx(tx_id):
-                raise AssertionError(
+                raise FailedToCreateDDO(
```

Comment:
something like web3 transaction reverted? etc.

Reply:
I didn't find something like that in the web3.exceptions but I am going to define it in the exception module.